src/Faker/ORM/PDO/Populator.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+namespace Faker\ORM\PDO;
+
+/**
+ * Service class for populating a database using the native PDO's.
+ */
+class Populator
+{
+    protected $generator;
+    protected $connection;
+    protected $rows = array();
+
+    /**
+     * @param \Faker\Generator $generator
+     * @param PDO|null $connection
+     */
+    public function __construct(\Faker\Generator $generator, \PDO $connection = null)
+    {
+        $this->generator = $generator;
+        $this->connection = $connection;
+    }
+
+    /**
+     * Use table name and provided mapping to build INSERT statement
+     * @return object PDOStatement object
+     */
+    private function prepareStatement($tableKey)
+    {
+        $result = null;
+        $columns = array();
+        $placeholders = array();
+
+        foreach ($this->rows[$tableKey][1] as $key => $value) {
+            $columns[] = $key;
+            $placeholders[] = ':' . $key;
+        }
+
+        $columns = implode(', ', $columns);
+        $placeholders = implode(', ', $placeholders);
+        $stmt = 'INSERT INTO '. trim($tableKey) .' (' . $columns . ') VALUES (' . $placeholders . ')';
+        $result = $this->connection->prepare($stmt);
+
+        return $result;
+    }
+
+    /**
+     * Convert the rows corresponding to the supplied table name to
+     * keys for use in pdo execute statement.
+     * @param  string $tableKey The table name this dataset is for.
+     * @return array            The array with placeholder keys
+     */
+    private function convertToPlaceholders($tableKey)
+    {
+        $originalArray = $this->rows[$tableKey];
+        $newArray = array();
+
+        for ($i=0; $i < count($originalArray); $i++) {
+            $newArray[$i] = array();
+            foreach ($originalArray[$i] as $key => $value) {
+                $newArray[$i][':'. $key] = $value;
+            }
+        }
+
+        return $newArray;
+    }
+
+    /**
+     * Return a faker value from the correct formatter.
+     * @param  string/array $request Data from supplied mapping
+     * @return string                Value returned by formatter or null
+     */
+    private function generateValue($request)
```

Comment:
`$request` doesn't mean anything here. Use the other populators as an example of naming.